addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -99,11 +99,15 @@ class MessageList extends Component {
             isLastMessageVisible:
                 lastMessageRef &&
                 lastMessageRef.isBottomVisible({ offset: 10 }),
-            scrollHeight: this.el.scrollHeight,
-            scrollTop: this.el.scrollTop,
+            scrollHeight: this.scrollableElement.scrollHeight,
+            scrollTop: this.scrollableElement.scrollTop,
         };
     }
 
+    mounted() {
+        this.scrollableElement = this._getScrollableElement();
+    }
+
```

Comment:
As I explained before, don't do this in mounted, the method call `_getScrollableElement()` must be dynamic.

Reply:
@seb-odoo . so you mean instead of using `this.scrollableElement`, just replace with `this._getScrollableElement()` everywhere right ?

Reply:
Yes